stubs/ext-ds.stub

```diff
@@ -153,16 +154,18 @@ final class Map implements IteratorAggregate, ArrayAccess, Collection
 	}
 
 	/**
-	 * @param Map<TKey, TValue> $map
-	 * @return Map<TKey, TValue>
+	 * @template TValue2
+	 * @param Map<TKey, TValue2> $map
+	 * @return Map<TKey, TValue|TValue2>
```

Comment:
This is wrong too. Since this is intersect the result values have to be `TValue&TValue2`

Reply:
Actually should stay as `TValue`
```php
     * @return Map A new map containing the pairs of the current instance
     *                 whose keys are also present in the given map. In other
     *                 words, returns a copy of the current map with all keys
     *                 removed that are not also in the other map.
     */
```